modes/bm/evil-collection-bm.el

```diff
@@ -0,0 +1,52 @@
+;;; evil-collection-bm.el --- Evil bindings for bm -*- lexical-binding: t -*-
+
+;; Copyright (C) 2017 James Nguyen
+
+;; Author: James Nguyen <james@jojojames.com>
```

Comment:
I suspect you are not James Nguyen ;)  Can you update the Copyright and Author lines accordingly?

Reply:
Sure. It's copied from other file